src/mixins/raster-layer-point-mixin.js

```diff
@@ -448,7 +448,44 @@ export default function rasterLayerPointMixin(_layer) {
     return _layer
   }
 
+  // used for setting LastFilteredSize when point map first initialized
+  let _tot = null
+  _layer.tot = function(number) {
+    if (!arguments.length) {
+      return _tot
+    }
+    _tot = number
+    return _layer
+  }
+
+  _layer.getTotalRecordsAsync = function(cf) {
+    if (_layer.tot()) {
+      return Promise.resolve()
+    }
+    return cf
+        .dimension()
```

Comment:
Wondering what the intent of this is - is it to just call this block of cf methods once and only once?

Reply:
It's a copy-paste for a pattern found in the count widget. I agree that it could be made simpler and more clear, for exemple by using the `tot` variable directly, as it doesn't need to be a layer method.